mullvad-daemon/src/lib.rs

```diff
@@ -1399,6 +1417,11 @@ where
         let _ = tx.send(save_result.map(|_| ()));
     }
 
+    fn handle_settings_changed(&mut self) {
+        self.event_listener
+            .notify_feature_indicators(self.get_feature_indicators())
+    }
```

Comment:
This seems a bit verbose to me. Should we really send a feature indicator nitification on every single settings change? Would it not be preferable to only send the event if an actual indicator was affected?

Reply:
Hmm, I'll see what I can do about it. Should be resolved by itself if `FeatureIndicators` is moved to `TunnelState`.

Reply:
Yup, `handle_settings_changed` could be removed :tada: